docs/api/covidcast.md

```diff
@@ -14,45 +14,136 @@ General topics not specific to any particular data source are discussed in the
+applications. These surveys are voluntary.
+
+The surveys are just one question long, and ask "Do you know someone in your
+community who is sick (fever, along with cough, or shortness of breath, or
+difficulty breathing) right now?" Using this survey data, we estimate the
 percentage of people in a given location, on a given day, that know somebody who
-has CLI (covid-like illness = fever, along with cough, or shortness of breath,
-or difficulty breathing).  Note that this is tracking a different quantity than
+has a COVID-like illness. Note that this is tracking a different quantity than
 the surveys through Facebook, and (unsurprisingly) the estimates here tend to be
 much larger.
-* `ght`: Data signal based on Google searches, provided to us by Google Health
-Trends.  Using this search data, we estimate the volume of covid-related
+
+| Signal | Description |
+| --- | --- |
+| `raw_cli` | Estimated fraction of people who know someone in their community with COVID-like illness |
+| `smoothed_cli` | Estimated fraction of people who know someone in their community with COVID-like illness, smoothed in time |
+
+#### `ght`
+
+Data signal based on Google searches, provided to us by Google Health
+Trends.  Using this search data, we estimate the volume of COVID-related
 searches in a given location, on a given day.  This signal is measured in
-arbitrary units (its scale is meaningless).
-* `quidel`: Data signal based on flu lab tests, provided to us by Quidel, Inc.
-When a patient (whether at a doctor’s office, clinic, or hospital) has
-covid-like symptoms, standard practice currently is to perform a flu test to
-rule out seasonal flu (influenza), because these two diseases have similar
-symptoms. Using this lab test data, we estimate the total number of flu tests
-per medical device (a measure of testing frequency), and the percentage of flu
-tests that are *negative* (since ruling out flu leaves open another
-cause---possibly covid---for the patient's symptoms), in a given location, 
-on a given day. 
-
-The data is expected to be updated daily. You can use the
-[`covidcast_meta`](covidcast_meta.md) endpoint to get summary information about
-the ranges of the different attributes for the different data sources currently
-in the data.
+arbitrary units (its scale is meaningless); larger numbers represent higher
+numbers of COVID-related searches.
+
+| Signal | Description |
+| --- | --- |
+| `raw_search` | Google search volume for COVID-related searches, in arbitrary units; normalized by population |
+| `smoothed_search` | Google search volume for COVID-related searches, in arbitrary units and normalized by population, smoothed in time using a Gaussian linear smoother |
+
+#### `doctor-visits`
+
+Data based on outpatient visits, provided to us by a national health system.
+Using this outpatient data, we estimate the percentage of COVID-related doctor's
+visits in a given location, on a given day.
```

Comment:
Not sure if worth commenting about the consistent 3 day delay we enforce.

Reply:
I've added a note saying we have lag, but not committing ourselves to exactly 3 days; the API will say what the latest available data is, anyway